### ISSUE TITLE
Players still receive viewable packets from despawned entities

### ISSUE THREAD

**Issue by Moulberry** [Moulberry/Graphite#10]
Entities which have been despawned due to player movement will still send packets to players (for 1 tick)

This can be fixed by only sending viewable packets if the chunk is within BOTH `chunk_view_position` and `new_chunk_view_position`

**Comment by GroobleDierne** [comment]
The comeback of Graphite's development?

**Comment by Moulberry** [comment]
> The comeback of Graphite's development?

No, sorry. I currently have other things I need to work on. Just leaving this issue here as a reminder for me when I come back.

**Comment by GroobleDierne** [comment]
You don't have to be sorry :smile: you are already kind enough to make your work open source.
